forte/pipeline.py

```diff
@@ -122,6 +125,7 @@ def __init__(self, resource: Optional[Resources] = None):
             self.resource = Resources()
         else:
             self.resource = resource
+        self.resource.update(onto_specs=ontology_file)
```

Comment:
So we are planning to have the ontology content in the pipeline for other purposes too. I think it is better to load the content of the `ontology_file` because the file path may not be accessible in all scenario (for example, in a distributed machine setting).